datadog_checks_base/datadog_checks/base/utils/db/core.py

```diff
@@ -209,3 +153,93 @@ def execute_query(self, query):
             return iter([])
 
         return chain((first_row,), rows)
+
+
+class QueryManager(QueryExecutor):
```

Comment:
While github shows all of this as green, none of this code is new, the only new code is the `super` call to the parent `QueryExecutor`